CHANGES.rst

```diff
@@ -11,6 +11,12 @@ datamodels
 
 - Update keyword comments/titles for V2_REF, V3_REF, FITXOFFS, FITYOFFS [#6822]
 
+extract_1d
+----------
+
+- Replace infinite values with NaNs, caused by zero division with
```

Comment:
Minor nit, but for the benefit of non-aficionados you could add that this change was made in order to fix a bug encountered in extraction of SOSS data.